merge-data.py

```diff
@@ -13,11 +17,19 @@
 output_dict = {item['issue']: item for item in gh_data_summary}
 
 def merge(dict1, dict2):
-    for key in dict1.keys():
-        if dict1[key]:
-            dict2[key] = dict1[key]
-    dict2.pop('issue', None)
+    """
+    Merges data from dict1 into dict2. Keys in dict1 take precedence if not None.
+    Strips trailing newline from 'description' and 'rationale'.
+    """
+    for key, value in dict1.items():
+        if value is not None:
+            if key in ['description', 'rationale'] and isinstance(value, str):
```

Comment:
Do you need to check for description/rationale?  Could this trim all strings?

Reply:
It could trim all strings.